client/landing/stepper/declarative-flow/entrepreneur-flow.ts

```diff
@@ -51,14 +50,7 @@ const entrepreneurFlow: Flow = {
 		const [ isMigrationFlow, setIsMigrationFlow ] = useState( false );
 
 		const getEntrepreneurLoginUrl = () => {
-			const queryParams = new URLSearchParams();
-
-			const redirectTo = addQueryArgs(
-				`${ window.location.protocol }//${ window.location.host }/setup/entrepreneur/trialAcknowledge`,
-				{
-					...Object.fromEntries( queryParams ),
-				}
-			);
+			const redirectTo = `${ window.location.protocol }//${ window.location.host }/setup/entrepreneur/trialAcknowledge${ window.location.search }`;
```

Comment:
This is just the simplify the code here (and fix an existing bug where `?ref=` is not inherited after redirection.